config/dev.exs

```diff
@@ -22,7 +22,7 @@ config :app, AppWeb.Endpoint,
   http: [ip: {127, 0, 0, 1}, port: 4000],
   check_origin: false,
   code_reloader: true,
-  debug_errors: true,
+  debug_errors: false,
```

Comment:
debugging in `dev` is actually desirable ... (IMO) 💭

Reply:
I agree, hence I was [reluctant to toggle it to `false`](https://github.com/dwyl/mvp/issues/269#issuecomment-1384336847).
However, in order for ErrorView to display `JSON` responses correctly, as it was requested in https://github.com/dwyl/mvp/issues/269, this config needs to be toggled off (as per docs -> https://hexdocs.pm/phoenix/custom_error_pages.html#the-errorview)

Reply:
https://github.com/dwyl/mvp/pull/270/files#diff-4f36118d05999cec87dec4895c8202881ba0577dc80fba4050aa50bbe9e326b3 (that was just merged) also had this change.